src/test/fixtures/editMode.ts

```diff
@@ -445,18 +451,27 @@ export const localeCreateBlueprintRequest: CreateBlueprintRequest = {
   },
 };
 
-export const timezoneBlueprintResponse: BlueprintResponse = {
-  ...timezoneCreateBlueprintRequest,
-  id: mockBlueprintIds['timezone'],
-  description: mockBlueprintDescriptions['timezone'],
-};
-
 export const localeBlueprintResponse: BlueprintResponse = {
   ...localeCreateBlueprintRequest,
   id: mockBlueprintIds['locale'],
   description: mockBlueprintDescriptions['locale'],
 };
 
+export const hostnameCreateBlueprintRequest: CreateBlueprintRequest = {
+  ...baseCreateBlueprintRequest,
+  name: mockBlueprintNames['hostname'],
+  description: mockBlueprintDescriptions['hostname'],
+  customizations: {
+    hostname: 'baseimage',
```

Comment:
wdyt maybe change the name to 
```suggestion
    hostname: 'baseImage',
```
or 
```suggestion
    hostname: 'base-image',
```

Reply:
Sure, the first suggestion might be invalid for a hostname, changed it to the second one.